packages/test-runner-playwright/src/index.ts

```diff
@@ -26,8 +26,20 @@ export function playwrightLauncher(args: PlaywrightLauncherArgs = {}) {
   const {
     product = 'chromium',
     launchOptions = {},
-    createBrowserContext = ({ browser }) => browser.newContext(),
-    createPage = ({ context }) => context.newPage(),
+    createBrowserContext = ({ browser }) => {
+      if ('browserWindow' in browser) {
+        return browser.context();
```

Comment:
There is no `newContext()` method on the `ElectronApplication` instance. There seems to be a single context for the whole `ElectronApplication` only, so maybe this is why the tests are not working?